instrumentation/github.com/emicklei/go-restful/otelrestful/example/server.go

```diff
@@ -1,6 +1,7 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
+// Package example exemplifies the otelrestful package.
```

Comment:
This is a program (command) and not a reusable package. The Go Doc Comment should be structured differently: https://tip.golang.org/doc/comment#cmd

```suggestion
// Example exemplifies the otelrestful package.
```